linux-support.md

```diff
@@ -1,38 +1,41 @@
 # .NET Support and Compatibility for Linux Distributions
 
-.NET is [supported by Microsoft](os-lifecycle-policy.md) and by [various commercial vendors and the community](linux.md) on multiple Linux distributions.
+.NET is supported by multiple [commercial and community organizations](support.md) on [multiple Linux distributions](linux.md).
```

Comment:
nit: should we not say multiple twice in a row?

Reply:
```suggestion
.NET is supported by various [commercial and community organizations](support.md) on [multiple Linux distributions](linux.md).
```